src/SDL2.cs

```diff
@@ -1673,6 +1732,11 @@ public unsafe struct SDL_RendererInfo
 			public fixed uint texture_formats[16];
 			public int max_texture_width;
 			public int max_texture_height;
+			public uint Format(int index) {
```

Comment:
It is if your main project is using C#, which has built-in support for arcane stuff like `fixed`.  This is not the case in every CLR language.